resources/progress_to_es.py

```diff
@@ -203,10 +203,22 @@ def main():
     parser.add_argument('-n', '--dry-run', default=False, action='store_true',
                         help='do not ingest into ES, just print')
     parser.add_argument('--log-level', default='info', choices=['debug', 'info', 'warning', 'error'])
+    parser.add_argument('--es_client_id',default=None,
+                        help='ES oauth2 client id')
+    parser.add_argument('--es_client_secret',default=None,
+                        help='ES oauth2 client secret')
+    parser.add_argument('--token_url',default=None,
+                        help='ES oauth2 realm token url')
     args = parser.parse_args()
 
     logging.basicConfig(level=getattr(logging, args.log_level.upper()), format='%(asctime)s %(levelname)s %(name)s : %(message)s')
 
+    es_api = ClientCredentialsAuth(address='https://elasticsearch.icecube.aq',
+                            token_url=args.token_url,
+                            client_secret=args.es_client_secret,
+                            client_id=args.es_client_id)
```

Comment:
One of the CI tests using flake8 is complaining about the under-indent here; line up the others on the same column as address.

Reply:
Ope, good catch updated!